lean/components/util/project_manager.py

```diff
@@ -157,6 +157,33 @@ def create_new_project(self, project_dir: Path, language: QCLanguage) -> None:
             self._generate_csproj(project_dir)
             self.generate_rider_config()
 
+    def delete_project(self, project_dir: Path) -> None:
+        """Deletes a project directory.
+
+        :cloud_projects: all projects fetched from the cloud
+        :param project_dir: the directory of the project to delete
+        """
+        shutil.rmtree(project_dir)
```

Comment:
Believe `shutil.rmtree` will throw if the path doesn't exist, we should handle this nicely I think, if it's not there just log?

nit - `:cloud_projects: all projects fetched from the cloud` shouldn't be there?